functests/1_performance/performance.go

```diff
@@ -231,7 +231,7 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 				Expect(sched_tasks).To(ContainSubstring("scheduling priority: 10"))
 			}
 		})
-		It("[test_id:42696][crit:medium][vendor:cnf-qe@redhat.com][level:acceptance] Stalld runs in higher priority than ksoftirq and rcu{c,b}", func() {
+		FIt("[test_id:42696][crit:medium][vendor:cnf-qe@redhat.com][level:acceptance] Stalld runs in higher priority than ksoftirq and rcu{c,b}", func() {
```

Comment:
Do we still want the Focus here?
Other than that, lgtm

Reply:
no we don't , just removed it , Thanks !